typescript/sdk/src/providers/transactions/submitter/TxSubmitterTypes.ts

```diff
@@ -1,5 +1,15 @@
+import { Address } from '@hyperlane-xyz/utils';
+
 export enum TxSubmitterType {
   JSON_RPC = 'JSON RPC',
   IMPERSONATED_ACCOUNT = 'Impersonated Account',
   GNOSIS_SAFE = 'Gnosis Safe',
 }
+
+export interface EV5GnosisSafeTxSubmitterProps {
+  safeAddress: Address;
```

Comment:
I do not think so, since `safeAddress` is definitely only used to set the safeAddress for proposals

With that said, I updated the below field on `EV5ImpersonatedAccountTxSubmitterProps` to be `userAddress` for symmetry (also more accurate)